api/java/src/main/java/com/qbot/answeringservice/config/WebSecurityConfig.java

```diff
@@ -13,7 +13,7 @@ public class WebSecurityConfig {
     @Bean
     public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
         http.authorizeRequests()
-                .antMatchers("/login", "/all", "/questions", "/actuator/**")
+                .antMatchers("/login", "/all", "/exchanges/**", "/actuator/**")
```

Comment:
@shravan-motipally should we update this with all of the new endpoints?

Reply:
Yea, ideally the should restrict all the current non-added endpoints.  If we can quickly add the new endpoints and also restrict everything else that would be great.  But all good if we don't want to spend time on that